composer.json

```diff
@@ -30,7 +30,7 @@
         "symfony/twig-bundle": "^3.4",
         "symfony/asset": "^3.4",
         "symfony/console": "^3.4",
-        "symfony/lts": "v3",
+        "symfony/symfony": "^3.4",
```

Comment:
```suggestion
        "symfony/symfony": "^3.4.25",
```
Same for symfony/phpunit-bridge

Reply:
Is it needed if composer.lock is up-to-date ?

Reply:
Yes because of security reason.
Always better to set our dependecy the most accurate